Userland/Applications/CertificateSettings/CertificateStore.cpp

```diff
@@ -65,13 +66,23 @@ GUI::Variant CertificateStoreModel::data(GUI::ModelIndex const& index, GUI::Mode
 
     auto cert = m_certificates.at(index.row());
 
+    auto issued_to = cert.subject.common_name();
+    if (issued_to.is_empty()) {
+        issued_to = cert.subject.organizational_unit();
+    }
+
+    auto issued_by = cert.issuer.common_name();
+    if (issued_by.is_empty()) {
+        issued_by = cert.issuer.organizational_unit();
+    }
```

Comment:
We are doing some unnesserary work here. Might be worth to put these into `{...}` blocks inside the `switch`?